src/OrchardCore.Modules/OrchardCore.Forms/Drivers/FormPartDisplay.cs

```diff
@@ -11,14 +12,27 @@ public class FormPartDisplay : ContentPartDisplayDriver<FormPart>
     {
         public override IDisplayResult Edit(FormPart part)
         {
-            return Initialize<FormPartEditViewModel>("FormPart_Fields_Edit", m =>
-            {
-                m.Action = part.Action;
-                m.Method = part.Method;
-                m.WorkflowTypeId = part.WorkflowTypeId;
-                m.EncType = part.EncType;
-                m.EnableAntiForgeryToken = part.EnableAntiForgeryToken;
-            });
+            var results = new List<IDisplayResult>();
+
+            results.Add(
+                Initialize<FormPartEditViewModel>("FormPart_Fields_Edit", m =>
+                {
+                    m.Action = part.Action;
+                    m.Method = part.Method;
+                    m.EncType = part.EncType;
+                    m.EnableAntiForgeryToken = part.EnableAntiForgeryToken;
+                })
+            );
+
+
+            results.Add(
+               Initialize<FormPartEditViewModel>("FormPart_WorkflowField_Edit", m =>
```

Comment:
This is part of the workflows module, so needs to be an extra driver in the workflows part. here it will appear even if workflows are not enabled